editorial_material/editorial_policies.tex

```diff
@@ -0,0 +1,84 @@
+\documentclass{article}
+\usepackage{graphicx}
+\usepackage{hyperref}
+
+\begin{document}
+
+\title{Editorial Policies of LiveCoMS}
+\author{Editorial Board of LiveCoMS}
+
+\maketitle
+
+\begin{abstract}
+The current draft list of editorial board policies for the Living
+Journal of Computational Molecular Science (LiveCoMS). These policies
+are not yet active, as they are still in draft.
+\end{abstract}
+
+\section{Policies on Submissions to LiveCoMS}
+
+\subsection{Policy on Licensing}
+
+LiveCoMS does not ask for or request any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distributed
+it free of charge. We recommend that the authors release the article
+under an open source license such as \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/},
+releasing the document for anyone to copy and redistribute the
+material in any medium or format, and remix, transform, and build upon
+the material for any purpose, even commercially.  Making it available
+to all of course makes it possible for LiveCoMS to publish it, and for
+the community to edit and contribute.
+
+Other more restrictive licenses may be possible as long as LiveCoMS
+has the permission to publish and excerpt from the document; this
+might be required if someone other than the authors has some rights to
+the material (for example, if it was previously published in some
+other journal). Generally, we only allow a more restrictive license in
+these cases, but are happy to discuss. Please ask the managing editors
+if you require some other licensing regime.
+
+\subsection{Policies on Author Order}
+Authors should, to the extent possible, determine the author order among themselves. 
```

Comment:
Perhaps add, "In general, each work should have a section describing the actual contributions of authors (and of those acknowledged) to provide clarity. However, we acknowledge that institutions evaluating for merit, promotion, tenure or other cases may not read this level of detail, so traditional notions of author order (first author, corresponding author, etc.) may still be relevant and the authors will need to coordinate who should occupy these positions."

Reply:
Done, though changed it from "should" to "must".  We can use the templates to provide guidance towards that.